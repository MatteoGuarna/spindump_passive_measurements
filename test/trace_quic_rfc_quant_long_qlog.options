--report-packets --report-notes --report-spins --format qlog
