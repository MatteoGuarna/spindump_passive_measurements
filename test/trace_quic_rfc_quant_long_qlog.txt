
The same test case and PCAP capture as in trace_quic_rfc_quant_long, but producing Qlog output.
