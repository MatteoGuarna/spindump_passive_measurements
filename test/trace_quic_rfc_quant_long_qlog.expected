{"qlog_version": "draft-02", "qlog_format": "JSON", "description": "Spindump measurements", "traces": [
 { "vantage_point": { "type": "network" }, "events": [
     {"time": 1614616215488286, "protocol_type": QUIC, "group_id": "sd-0", "event": "connection_started", 
      "ip_version": "ipv4", "src_ip": "10.30.0.167", "dst_ip": "91.190.195.94", "src_port": "49702", "dst_port": "4433", "src_cid": "041d0cd3", "dst_cid": "c2e9970d6b5925175646", 
      "data": {"packets1": 0, "packets2": 0, "bytes1": 0, "bytes2": 0}},
     {"time": 1614616216489070, "protocol_type": QUIC, "group_id": "sd-0", "event": "measurement", 
      "data": {"packets1": 2, "packets2": 0, "bytes1": 2728, "bytes2": 0, "bandwidth1": 1500, "bandwidth2": 0, "Dir": "initiator", "Length": 1228}},
     {"time": 1614616216598870, "protocol_type": QUIC, "group_id": "sd-0", "event": "measurement", 
      "data": {"packets1": 2, "packets2": 0, "bytes1": 2728, "bytes2": 0, "bandwidth1": 1500, "bandwidth2": 0, "value": 0, "Who": "responder"}},
     {"time": 1614616216598876, "protocol_type": QUIC, "group_id": "sd-0", "event": "connection_started", 
      "ip_version": "ipv4", "src_ip": "10.30.0.167", "dst_ip": "91.190.195.94", "src_port": "49702", "dst_port": "4433", "src_cid": "041d0cd3", "dst_cid": "0487c887", 
      "data": {"packets1": 2, "packets2": 1, "bytes1": 2728, "bytes2": 298, "bandwidth1": 1500, "bandwidth2": 298}},
     {"time": 1614616216598876, "protocol_type": QUIC, "group_id": "sd-0", "event": "measurement", 
      "data": {"packets1": 2, "packets2": 1, "bytes1": 2728, "bytes2": 298, "bandwidth1": 1500, "bandwidth2": 298, "right_rtt": 109806}},
     {"time": 1614616216600724, "protocol_type": QUIC, "group_id": "sd-0", "event": "measurement", 
      "data": {"packets1": 3, "packets2": 2, "bytes1": 2856, "bytes2": 1798, "bandwidth1": 1500, "bandwidth2": 1798, "Dir": "initiator", "Length": 128}},
     {"time": 1614616216663302, "protocol_type": QUIC, "group_id": "sd-0", "event": "measurement", 
      "data": {"packets1": 3, "packets2": 2, "bytes1": 2856, "bytes2": 1798, "bandwidth1": 1500, "bandwidth2": 1798, "value": 0, "Who": "responder"}},
     {"time": 1614616216663306, "protocol_type": QUIC, "group_id": "sd-0", "event": "measurement", 
      "data": {"packets1": 3, "packets2": 3, "bytes1": 2856, "bytes2": 3078, "bandwidth1": 1500, "bandwidth2": 3078, "value": 0, "Who": "responder"}},
     {"time": 1614616216663307, "protocol_type": QUIC, "group_id": "sd-0", "event": "measurement", 
      "data": {"packets1": 3, "packets2": 4, "bytes1": 2856, "bytes2": 4358, "bandwidth1": 1500, "bandwidth2": 4358, "value": 0, "Who": "responder"}},
     {"time": 1614616216825651, "protocol_type": QUIC, "group_id": "sd-0", "event": "measurement", 
      "data": {"packets1": 3, "packets2": 5, "bytes1": 2856, "bytes2": 5638, "bandwidth1": 1500, "bandwidth2": 5638, "value": 0, "Who": "responder"}},
     {"time": 1614616216825655, "protocol_type": QUIC, "group_id": "sd-0", "event": "measurement", 
      "data": {"packets1": 3, "packets2": 6, "bytes1": 2856, "bytes2": 6918, "bandwidth1": 1500, "bandwidth2": 6918, "value": 0, "Who": "responder"}},
     {"time": 1614616216825657, "protocol_type": QUIC, "group_id": "sd-0", "event": "measurement", 
      "data": {"packets1": 3, "packets2": 7, "bytes1": 2856, "bytes2": 8198, "bandwidth1": 1500, "bandwidth2": 8198, "value": 0, "Who": "responder"}},
     {"time": 1614616216825658, "protocol_type": QUIC, "group_id": "sd-0", "event": "measurement", 
      "data": {"packets1": 3, "packets2": 8, "bytes1": 2856, "bytes2": 9478, "bandwidth1": 1500, "bandwidth2": 9478, "value": 0, "Who": "responder"}},
     {"time": 1614616216825660, "protocol_type": QUIC, "group_id": "sd-0", "event": "measurement", 
      "data": {"packets1": 3, "packets2": 9, "bytes1": 2856, "bytes2": 10758, "bandwidth1": 1500, "bandwidth2": 10758, "value": 0, "Who": "responder"}},
     {"time": 1614616216825662, "protocol_type": QUIC, "group_id": "sd-0", "event": "measurement", 
      "data": {"packets1": 3, "packets2": 10, "bytes1": 2856, "bytes2": 12038, "bandwidth1": 1500, "bandwidth2": 12038, "value": 0, "Who": "responder"}},
     {"time": 1614616216844735, "protocol_type": QUIC, "group_id": "sd-0", "event": "measurement", 
      "data": {"packets1": 3, "packets2": 11, "bytes1": 2856, "bytes2": 13318, "bandwidth1": 1500, "bandwidth2": 13318, "value": 0, "Who": "initiator"}},
     {"time": 1614616216844735, "protocol_type": QUIC, "group_id": "sd-0", "event": "measurement", 
      "data": {"packets1": 4, "packets2": 11, "bytes1": 2960, "bytes2": 13318, "bandwidth1": 1500, "bandwidth2": 13318, "value": 0, "Who": "initiator"}},
     {"time": 1614616216910314, "protocol_type": QUIC, "group_id": "sd-0", "event": "measurement", 
      "data": {"packets1": 5, "packets2": 11, "bytes1": 3013, "bytes2": 13318, "bandwidth1": 1500, "bandwidth2": 13318, "value": 0, "Who": "responder"}},
     {"time": 1614616216911013, "protocol_type": QUIC, "group_id": "sd-0", "event": "measurement", 
      "data": {"packets1": 6, "packets2": 12, "bytes1": 3095, "bytes2": 13568, "bandwidth1": 1500, "bandwidth2": 13568, "Dir": "initiator", "Length": 82}},
     {"time": 1614616217105839, "protocol_type": QUIC, "group_id": "sd-0", "event": "measurement", 
      "data": {"packets1": 6, "packets2": 12, "bytes1": 3095, "bytes2": 13568, "bandwidth1": 1500, "bandwidth2": 13568, "value": 0, "Who": "responder"}},
     {"time": 1614616217105847, "protocol_type": QUIC, "group_id": "sd-0", "event": "measurement", 
      "data": {"packets1": 6, "packets2": 13, "bytes1": 3095, "bytes2": 14920, "bandwidth1": 1500, "bandwidth2": 14920, "value": 0, "Who": "responder"}},
     {"time": 1614616217105849, "protocol_type": QUIC, "group_id": "sd-0", "event": "measurement", 
      "data": {"packets1": 6, "packets2": 14, "bytes1": 3095, "bytes2": 16200, "bandwidth1": 1500, "bandwidth2": 16200, "value": 0, "Who": "responder"}},
     {"time": 1614616217105851, "protocol_type": QUIC, "group_id": "sd-0", "event": "measurement", 
      "data": {"packets1": 6, "packets2": 15, "bytes1": 3095, "bytes2": 17480, "bandwidth1": 1500, "bandwidth2": 17480, "value": 0, "Who": "responder"}},
     {"time": 1614616217105853, "protocol_type": QUIC, "group_id": "sd-0", "event": "measurement", 
      "data": {"packets1": 6, "packets2": 16, "bytes1": 3095, "bytes2": 18760, "bandwidth1": 1500, "bandwidth2": 18760, "value": 0, "Who": "responder"}},
     {"time": 1614616217105855, "protocol_type": QUIC, "group_id": "sd-0", "event": "measurement", 
      "data": {"packets1": 6, "packets2": 17, "bytes1": 3095, "bytes2": 20040, "bandwidth1": 1500, "bandwidth2": 20040, "value": 0, "Who": "responder"}},
     {"time": 1614616217107189, "protocol_type": QUIC, "group_id": "sd-0", "event": "measurement", 
      "data": {"packets1": 6, "packets2": 18, "bytes1": 3095, "bytes2": 21320, "bandwidth1": 1500, "bandwidth2": 21320, "value": 1, "Who": "initiator"}},
     {"time": 1614616217177621, "protocol_type": QUIC, "group_id": "sd-0", "event": "measurement", 
      "data": {"packets1": 7, "packets2": 18, "bytes1": 3159, "bytes2": 21320, "bandwidth1": 1500, "bandwidth2": 21320, "value": 1, "Who": "responder"}},
     {"time": 1614616217177621, "protocol_type": QUIC, "group_id": "sd-0", "event": "measurement", 
      "data": {"packets1": 7, "packets2": 18, "bytes1": 3159, "bytes2": 21320, "bandwidth1": 1500, "bandwidth2": 21320, "right_rtt": 70432}},
     {"time": 1614616217178198, "protocol_type": QUIC, "group_id": "sd-0", "event": "measurement", 
      "data": {"packets1": 7, "packets2": 19, "bytes1": 3159, "bytes2": 21415, "bandwidth1": 1500, "bandwidth2": 21415, "value": 0, "Who": "initiator"}},
     {"time": 1614616217178198, "protocol_type": QUIC, "group_id": "sd-0", "event": "measurement", 
      "data": {"packets1": 7, "packets2": 19, "bytes1": 3159, "bytes2": 21415, "bandwidth1": 1500, "bandwidth2": 21415, "left_rtt": 577}},
     {"time": 1614616217178198, "protocol_type": QUIC, "group_id": "sd-0", "event": "measurement", 
      "data": {"packets1": 7, "packets2": 19, "bytes1": 3159, "bytes2": 21415, "bandwidth1": 1500, "bandwidth2": 21415, "full_rtt_initiator": 71009}},
     {"time": 1614616217367009, "protocol_type": QUIC, "group_id": "sd-0", "event": "measurement", 
      "data": {"packets1": 8, "packets2": 19, "bytes1": 3217, "bytes2": 21415, "bandwidth1": 1500, "bandwidth2": 21415, "value": 1, "Who": "responder"}},
     {"time": 1614616217367012, "protocol_type": QUIC, "group_id": "sd-0", "event": "measurement", 
      "data": {"packets1": 8, "packets2": 20, "bytes1": 3217, "bytes2": 22695, "bandwidth1": 1500, "bandwidth2": 22695, "value": 1, "Who": "responder"}},
     {"time": 1614616217367013, "protocol_type": QUIC, "group_id": "sd-0", "event": "measurement", 
      "data": {"packets1": 8, "packets2": 21, "bytes1": 3217, "bytes2": 23975, "bandwidth1": 1500, "bandwidth2": 23975, "value": 1, "Who": "responder"}},
     {"time": 1614616217367014, "protocol_type": QUIC, "group_id": "sd-0", "event": "measurement", 
      "data": {"packets1": 8, "packets2": 22, "bytes1": 3217, "bytes2": 25255, "bandwidth1": 1500, "bandwidth2": 25255, "value": 1, "Who": "responder"}},
     {"time": 1614616217367015, "protocol_type": QUIC, "group_id": "sd-0", "event": "measurement", 
      "data": {"packets1": 8, "packets2": 23, "bytes1": 3217, "bytes2": 26535, "bandwidth1": 1500, "bandwidth2": 26535, "value": 1, "Who": "responder"}},
     {"time": 1614616217367015, "protocol_type": QUIC, "group_id": "sd-0", "event": "measurement", 
      "data": {"packets1": 8, "packets2": 24, "bytes1": 3217, "bytes2": 27815, "bandwidth1": 1500, "bandwidth2": 27815, "value": 1, "Who": "responder"}},
     {"time": 1614616217367017, "protocol_type": QUIC, "group_id": "sd-0", "event": "measurement", 
      "data": {"packets1": 8, "packets2": 25, "bytes1": 3217, "bytes2": 29095, "bandwidth1": 1500, "bandwidth2": 29095, "value": 1, "Who": "responder"}},
     {"time": 1614616217367629, "protocol_type": QUIC, "group_id": "sd-0", "event": "measurement", 
      "data": {"packets1": 9, "packets2": 26, "bytes1": 3275, "bytes2": 30375, "bandwidth1": 1500, "bandwidth2": 30375, "Dir": "initiator", "Length": 58}},
     {"time": 1614616217545056, "protocol_type": QUIC, "group_id": "sd-0", "event": "measurement", 
      "data": {"packets1": 9, "packets2": 26, "bytes1": 3275, "bytes2": 30375, "bandwidth1": 1500, "bandwidth2": 30375, "value": 0, "Who": "responder"}},
     {"time": 1614616217545056, "protocol_type": QUIC, "group_id": "sd-0", "event": "measurement", 
      "data": {"packets1": 9, "packets2": 26, "bytes1": 3275, "bytes2": 30375, "bandwidth1": 1500, "bandwidth2": 30375, "right_rtt": 366858}},
     {"time": 1614616217545056, "protocol_type": QUIC, "group_id": "sd-0", "event": "measurement", 
      "data": {"packets1": 9, "packets2": 26, "bytes1": 3275, "bytes2": 30375, "bandwidth1": 1500, "bandwidth2": 30375, "full_rtt_responder": 367435}},
     {"time": 1614616217545062, "protocol_type": QUIC, "group_id": "sd-0", "event": "measurement", 
      "data": {"packets1": 9, "packets2": 27, "bytes1": 3275, "bytes2": 31655, "bandwidth1": 1500, "bandwidth2": 31655, "value": 0, "Who": "responder"}},
     {"time": 1614616217545065, "protocol_type": QUIC, "group_id": "sd-0", "event": "measurement", 
      "data": {"packets1": 9, "packets2": 28, "bytes1": 3275, "bytes2": 32935, "bandwidth1": 1500, "bandwidth2": 32935, "value": 0, "Who": "responder"}},
     {"time": 1614616217545067, "protocol_type": QUIC, "group_id": "sd-0", "event": "measurement", 
      "data": {"packets1": 9, "packets2": 29, "bytes1": 3275, "bytes2": 34215, "bandwidth1": 1500, "bandwidth2": 34215, "value": 0, "Who": "responder"}},
     {"time": 1614616217546034, "protocol_type": QUIC, "group_id": "sd-0", "event": "measurement", 
      "data": {"packets1": 9, "packets2": 30, "bytes1": 3275, "bytes2": 34607, "bandwidth1": 1500, "bandwidth2": 34607, "value": 1, "Who": "initiator"}},
     {"time": 1614616217546034, "protocol_type": QUIC, "group_id": "sd-0", "event": "measurement", 
      "data": {"packets1": 9, "packets2": 30, "bytes1": 3275, "bytes2": 34607, "bandwidth1": 1500, "bandwidth2": 34607, "left_rtt": 978}},
     {"time": 1614616217546034, "protocol_type": QUIC, "group_id": "sd-0", "event": "measurement", 
      "data": {"packets1": 9, "packets2": 30, "bytes1": 3275, "bytes2": 34607, "bandwidth1": 1500, "bandwidth2": 34607, "full_rtt_initiator": 367836}},
     {"time": 1614616217547241, "protocol_type": QUIC, "group_id": "sd-0", "event": "measurement", 
      "data": {"packets1": 10, "packets2": 30, "bytes1": 3333, "bytes2": 34607, "bandwidth1": 1775, "bandwidth2": 34607, "value": 1, "Who": "initiator"}},
     {"time": 1614616217643280, "protocol_type": QUIC, "group_id": "sd-0", "event": "measurement", 
      "data": {"packets1": 11, "packets2": 30, "bytes1": 3386, "bytes2": 34607, "bandwidth1": 1775, "bandwidth2": 34607, "value": 1, "Who": "responder"}},
     {"time": 1614616217643280, "protocol_type": QUIC, "group_id": "sd-0", "event": "measurement", 
      "data": {"packets1": 11, "packets2": 30, "bytes1": 3386, "bytes2": 34607, "bandwidth1": 1775, "bandwidth2": 34607, "right_rtt": 97246}},
     {"time": 1614616217643280, "protocol_type": QUIC, "group_id": "sd-0", "event": "measurement", 
      "data": {"packets1": 11, "packets2": 30, "bytes1": 3386, "bytes2": 34607, "bandwidth1": 1775, "bandwidth2": 34607, "full_rtt_responder": 98224}},
     {"time": 1614616217643523, "protocol_type": QUIC, "group_id": "sd-0", "event": "measurement", 
      "data": {"packets1": 12, "packets2": 31, "bytes1": 3447, "bytes2": 34666, "bandwidth1": 1775, "bandwidth2": 34607, "Dir": "initiator", "Length": 61}},
     {"time": 1614616217690841, "protocol_type": QUIC, "group_id": "sd-0", "event": "measurement", 
      "data": {"packets1": 12, "packets2": 31, "bytes1": 3447, "bytes2": 34666, "bandwidth1": 1775, "bandwidth2": 34607, "value": 1, "Who": "responder"}}  
 ]}]}
